cmd/restic/cmd_backup.go

```diff
@@ -113,7 +113,7 @@ func init() {
 	f.BoolVar(&backupOptions.ExcludeCaches, "exclude-caches", false, `excludes cache directories that are marked with a CACHEDIR.TAG file. See http://bford.info/cachedir/spec.html for the Cache Directory Tagging Standard`)
 	f.BoolVar(&backupOptions.Stdin, "stdin", false, "read backup from stdin")
 	f.StringVar(&backupOptions.StdinFilename, "stdin-filename", "stdin", "`filename` to use when reading from stdin")
-	f.StringArrayVar(&backupOptions.Tags, "tag", nil, "add a `tag` for the new snapshot (can be specified multiple times)")
+	f.Var(&backupOptions.Tags, "tag", "add a `tag` for the new snapshot in the format `tag[,tag,...]` (can be specified multiple times)")
```

Comment:
With this change you actually add tags to the new snapshot.

Reply:
You mean just for the help message ? With a `s` ?

Reply:
Yes. my idea was to replace "a \`tag\`" with \`tags\`